UploadDaemon/UploadDaemon.csproj

```diff
@@ -106,8 +106,8 @@
     <Compile Include="Archiving\IDateTimeProvider.cs" />
     <Compile Include="Configuration\Config.cs" />
     <Compile Include="Configuration\ConfigParser.cs" />
-    <Compile Include="LineCoverageMerger.cs" />
-    <Compile Include="MessageFormatter.cs" />
+    <Compile Include="Upload\LineCoverageMerger.cs" />
+    <Compile Include="Utils\MessageFormatter.cs" />
```

Comment:
I moved the GlobPatternList to Configuration since it's used there as well and I didn't want a dependency from Config to SymbolAnalysis, but otherwise implemented your changes as suggested